mybatis-spring-boot-autoconfigure/src/test/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfigurationTest.java

```diff
@@ -164,6 +167,69 @@ public void testWithDatabaseIdProvider() {
 		assertEquals("h2", this.context.getBean(SqlSessionFactory.class).getConfiguration().getDatabaseId());
 	}
 
+	@Test
+	public void testMixedConfigurations() {
+		// set up
+		{
```

Comment:
I've added some test cases via 5e2c08ad20d767a940c2ecfc745a7335fbcc7702.
Is OK ?